channels/rail/client/rail_orders.c

```diff
@@ -811,6 +978,24 @@ UINT rail_order_recv(railPlugin* rail, wStream* s)
 				return rail_recv_langbar_info_order(rail, &langBarInfo, s);
 			}
 
+		case RDP_RAIL_ORDER_ZORDER_SYNC:
+			{
+				RAIL_ZORDER_ZYNC val;
```

Comment:
Typo: `RAIL_ZORDER_ZYNC` should be `RAIL_ZORDER_SYNC`

Reply:
@mfleisz done